### DISCUSSION TITLE
Banking app recently got updated and shamiko no longer works on it

### DISCUSSION THREAD

**Discussion by Sygmstr** [LSPosed/LSPosed#2166] [Q&A]
Recently I updated my banking app NLB pay and it keeps crashing. 
![Screenshot_20221005_185738_Google Play services](https://user-images.githubusercontent.com/96848770/194751915-755b7095-c264-4d96-b2dc-84167a1604d7.jpg)
![Screenshot_20221005_185806_Google Play services](https://user-images.githubusercontent.com/96848770/194751928-d5330ea9-11c9-4003-8bd0-6d03bc6f02f1.jpg)
Anyone know how I can bypass?

**Comment by Okda4ever** [comment]
the listed below banking app was updated yesterday and also stopped working, seems they found out a new way to detect root

https://play.google.com/store/apps/details?id=paymob.cib.smartwallet

**Reply by Okda4ever** [reply]
it doesn't crash , it only stats that the mobile is not secure, how can i help ?

**Reply by Okda4ever** [reply]
another banking application updated today refuse to work and detect roots

https://apkcombo.com/instapay-egypt/com.egyptianbanks.instapay/old-versions/

version 1.5.0 is the version i am talking about

1.4.5 is working normally

**Reply by mylove90** [reply]
> another banking application updated today refuse to work and detect roots
> 
> https://apkcombo.com/instapay-egypt/com.egyptianbanks.instapay/old-versions/
> 
> version 1.5.0 is the version i am talking about
> 
> 1.4.5 is working normally

That same app was working on unrooted custom rom till version 1.5.3...now starting from version 1.5.4 somehow it detects custom roms too and doesn't work on any device with a custom rom regardless it is being rooted or not

**Reply by NEO701** [reply]
Any workaround found for Instapay?
